pkg/gitreceive/k8s_util_test.go

```diff
@@ -0,0 +1,130 @@
+package gitreceive
+
+import (
+	// "reflect"
+	"fmt"
+	"strings"
+	"testing"
+
+	"k8s.io/kubernetes/pkg/api"
+)
+
+func DockerBuilderPodName(t *testing.T) {
+	name := dockerBuilderPodName("demo", "12345678")
+	if !strings.HasPrefix(name, "dockerbuild-demo-12345678-") {
+		t.Fatalf("expected pod name dockerbuild-demo-12345678-, got %s", name)
+	}
+}
+
+func SlugBuilderPodName(t *testing.T) {
+	name := dockerBuilderPodName("demo", "12345678")
+	if !strings.HasPrefix(name, "slugbuild-demo-12345678-") {
+		t.Fatalf("expected pod name slugbuild-demo-12345678-, got %s", name)
+	}
+}
```

Comment:
this and the above `func` names need to start with `Test`, or they won't be executed as test cases